moveit_ros/move_group/src/move_group.cpp

```diff
@@ -38,6 +38,8 @@
 #include <tf2_ros/transform_listener.h>
 #include <moveit/move_group/capability_names.h>
 #include <moveit/move_group/move_group_capability.h>
+#include <moveit/move_group/move_group_context.h>
```

Comment:
These two includes seem redundant. I guess they were removed with the sync but then just added again with the rebase.

Reply:
removed redundant 44c1643